# Copyright (c) 2026 The Mogu Authors.
# All rights reserved.
#
# Enable ENCODE + BMP for legacy_render Save2Image. Leave JPG/PNG/etc. off:
# this tree's CxImage/ subdir may be a junction whose ../jpeg does not resolve.
--- a/CxImage/ximacfg.h
+++ b/CxImage/ximacfg.h
@@ -3,18 +3,18 @@
 
 /////////////////////////////////////////////////////////////////////////////
 // CxImage supported features
-#define CXIMAGE_SUPPORT_ALPHA          0
-#define CXIMAGE_SUPPORT_SELECTION      0
-#define CXIMAGE_SUPPORT_TRANSFORMATION 0
-#define CXIMAGE_SUPPORT_DSP            0
-#define CXIMAGE_SUPPORT_LAYERS		   0
-#define CXIMAGE_SUPPORT_INTERPOLATION  0
+#define CXIMAGE_SUPPORT_ALPHA          1
+#define CXIMAGE_SUPPORT_SELECTION      1
+#define CXIMAGE_SUPPORT_TRANSFORMATION 1
+#define CXIMAGE_SUPPORT_DSP            1
+#define CXIMAGE_SUPPORT_LAYERS		   1
+#define CXIMAGE_SUPPORT_INTERPOLATION  1
 
 #define CXIMAGE_SUPPORT_DECODE	1
-#define CXIMAGE_SUPPORT_ENCODE	0		//<vho><T.Peck>
+#define CXIMAGE_SUPPORT_ENCODE	1
 #define	CXIMAGE_SUPPORT_WINDOWS 1
 #define	CXIMAGE_SUPPORT_EXIF    0
 
 /////////////////////////////////////////////////////////////////////////////
 // CxImage supported formats
-#define CXIMAGE_SUPPORT_BMP 0
+#define CXIMAGE_SUPPORT_BMP 1
 #define CXIMAGE_SUPPORT_GIF 1
 #define CXIMAGE_SUPPORT_JPG 0
 #define CXIMAGE_SUPPORT_PNG 0
